dipy/reconst/dki_micro.py

```diff
@@ -0,0 +1,647 @@
+        return _compartments_eigenvalues(self.model_params[..., 28:34],
+                                         self._min_diffusivity)
+
+    @property
+    def axonal_diffusivity(self):
+        """ Returns the axonal diffusivity defined as the restricted diffusion
+        tensor trace [1]_.
+
+        References
+        ----------
+        .. [1] Fieremans, E., Jensen, J.H., Helpern, J.A., 2011. White Matter
+               Characterization with Diffusion Kurtosis Imaging. Neuroimage
+               58(1): 177-188. doi:10.1016/j.neuroimage.2011.06.006
+        """
+        return trace(self.restricted_evals)
+
+    @property
+    def hindered_ad(self):
+        """ Returns the axial diffusivity of the hindered compartment.
+
+        Note
+        -----
+        The hindered diffusion tensor can be seem as the tissue's
+        extra-cellular diffusion compartment [1]_.
+
+        References
+        ----------
+        .. [1] Fieremans, E., Jensen, J.H., Helpern, J.A., 2011. White Matter
+               Characterization with Diffusion Kurtosis Imaging. Neuroimage
+               58(1): 177-188. doi:10.1016/j.neuroimage.2011.06.006
+        """
+        return axial_diffusivity(self.hindered_evals)
+
+    @property
+    def hindered_rd(self):
+        """ Returns the radial diffusivity of the hindered compartment.
+
+        Note
+        -----
+        The hindered diffusion tensor can be seem as the tissue's
+        extra-cellular diffusion compartment [1]_.
+
+        References
+        ----------
+        .. [1] Fieremans, E., Jensen, J.H., Helpern, J.A., 2011. White Matter
+               Characterization with Diffusion Kurtosis Imaging. Neuroimage
+               58(1): 177-188. doi:10.1016/j.neuroimage.2011.06.006
+        """
+        return radial_diffusivity(self.hindered_evals)
+
+    @property
+    def tortuosity(self):
+        """ Returns the tortuosity of the hindered diffusion which is defined
+        by ADe / RDe, where ADe and RDe are the axial and radial diffusivities
+        of the hindered compartment [1]_.
+
+        Note
+        -----
+        The hindered diffusion tensor can be seem as the tissue's
+        extra-cellular diffusion compartment [1]_.
+
+        References
+        ----------
+        .. [1] Fieremans, E., Jensen, J.H., Helpern, J.A., 2011. White Matter
+               Characterization with Diffusion Kurtosis Imaging. Neuroimage
+               58(1): 177-188. doi:10.1016/j.neuroimage.2011.06.006
+        """
+        return tortuosity(self.hindered_ad, self.hindered_rd,
+                          min_diffusivity=self._min_diffusivity)
+
+    def _is_awfonly(self):
+        """ To raise error if only the axonal water fraction was computed """
+        if self.model_params.shape[-1] < 39:
+            raise ValueError('Only the awf was processed! Rerun model fit '
+                             'with input parameter awf_only set to False')
+
+    @property
+    def _min_diffusivity(self):
```

Comment:
Maybe move this function to `dti`? You should still have it here, because of the inheritance.